db/migrate/20240216163519_add_no_int_fix_marker_to_bulk_uploads.rb

```diff
@@ -0,0 +1,5 @@
+class AddNoIntFixMarkerToBulkUploads < ActiveRecord::Migration[7.0]
+  def change
+    add_column :bulk_uploads, :noint_fix_status, :string, default: "not_applied"
```

Comment:
Would the default get added to existing records? If no, wouldn't think make more sense as `not_needed`?

Reply:
To confirm, yes it does get added to existing records.